plugins/inventory/now.py

```diff
@@ -25,6 +25,7 @@
 version_added: 1.0.0
 extends_documentation_fragment:
   - ansible.builtin.constructed
+  - inventory_cache
 notes:
   - Query feature and constructed groups were added in version 1.2.0.
```

Comment:
```suggestion
  - Caching was added in version 2.5.0.
```

Reply:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
Add integration tests for this change so that we make sure that we don't break anything.